lib/models/case.js

```diff
@@ -47,6 +47,13 @@ class Case {
     });
   }
 
+  patch(data, opts) {
+    return Case.query().findById(this.id)
+      .then(result => {
+        return this.update({ ...result.data, ...data }, opts);
+      });
+  }
```

Comment:
I would've done a `Case.query().patchAndFetchById(id, data)` but not sure how opts fit in

Reply:
That's what `update` already does. It shallow-patches the whole model though, and not the properties in `data`. So the whole `data` object is replaced by the new `data`.

Reply:
aaah

Reply:
`deepPatchAndFetchById` then :trollface: